share/commands/agent-inventory

```diff
@@ -34,7 +34,7 @@ bootstrap_check
 # We cannnot start an inventory in bootstrap policies
 if cmp -s "/var/rudder/cfengine-community/inputs/failsafe.cf" "/opt/rudder/share/bootstrap-promises/failsafe.cf"
 then
-  >&2 echo "Agent is currently in bootstrap policies, cannot run an inventory. Please download initial policies from the server with command `rudder agent update` first. If problem persists, use `rudder agent check` for diagnostic"
+  >&2 echo "Agent is currently in bootstrap policies, cannot run an inventory. Please download initial policies from the server with command \`rudder agent update\` first. If problem persists, use \`rudder agent check\` for diagnostic"
```

Comment:
just don't use backticks in shell code and put simple quotes instead

Reply:
I agree, it's less dangerous